src/test/container.go

```diff
@@ -117,3 +119,19 @@ func retrieveFile(target *core.BuildTarget, cid []byte, filename string, warn bo
 		}
 	}
 }
+
+// runPlzSandboxedTest runs a test using please_sandbox, which simply restricts network & process
+// namespaces. It's only available on Linux and must be installed separately, since it has to be
+// suid root in order to use CLONE_NEWNET.
+func runPlzSandboxedTest(state *core.BuildState, target *core.BuildTarget) ([]byte, error) {
```

Comment:
This comment is actually slightly out of date now (because it also restricts PID, IPC and UTS as well, for example).
Can look at the options around FS; I think we can also increase the level of sandboxing over time.
Unsure about UID. Docker caused many issues with that by creating files that were owned by root and couldn't be subsequently deleted. Would make sense, but only if that can be avoided.